projects/fsastorefront-e2e-cypress/cypress/helpers/checkout/insurance/travel-checkout.ts

```diff
@@ -160,3 +160,58 @@ export function checkPageComponenth3(checkoutStep) {
       cy.get('h2').contains('Travel Insurance');
     });
 }
+
+export function checkBackpackersTravelComparisonTable() {
+  cy.get('cx-fs-comparison-table-container').within(() => {
+    cy.get('.nav-link').contains(' Backpacking Trip').click();
```

Comment:
`.should('contain.text'` instead of `contains`

Reply:
Tried with should, not working in that case. Space is removed